html/semantics/forms/textfieldselection/selection-after-content-change.html

```diff
@@ -92,26 +92,49 @@
   }, `${descriptor}: selection must change when setting a different value`);
 }
 
-test(() => {
-  const textarea = document.querySelector("#t1");
-  textarea.value = "hell\no";
-  textarea.setSelectionRange(1, 3, "backward");
+function runInputTest(descriptor, elementFactory) {
+  runTest(descriptor, elementFactory);
+
+  test(() => {
```

Comment:
This test fails, because in the "input in document" case elementFactory just keeps returning the same node without doing anything to its value.  runTest will set the value to "hello", then to "hello2".  Then this test will run and set it to a value that sanitizes to "hello", which is not the same as "hello2", so the cursor will get moved.